### PR TITLE
Add CRA React15 example to test back-compat

### PR DESCRIPTION
**Pull request by shilman**
Issue: #6474

## What I did

Added an example using react15 to the repo to demonstrate a problem with theming

## How to test

```
yarn bootstrap --reset --core
cd examples/creact-cra15
yarn storybook
```

Note that if you run this example using `5.0.x` and DON'T theme, it seems to work OK

### CODE REVIEW

File: examples/cra-react15/.storybook/config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { configure, addParameters } from '@storybook/react';
+import { create } from '@storybook/theming';
+
+function loadStories() {
+  require('../src/stories');
+}
+
+addParameters({
+  options: {
+    theme: create({ colorPrimary: 'hotpink', colorSecondary: 'orangered' }),
```
Comment (tmeasday) [inline examples/cra-react15/.storybook/config.js]: Do you think you should comment out the theme to stop this from breaking `next`? Will still make it much easier to test

Comment (shilman) [reply examples/cra-react15/.storybook/config.js]: @tmeasday Unfortunately it still breaks, I think due to #6435

**Comment by ndelangen** [comment]
@shilman please take a look it should work now

File: addons/storyshots/storyshots-core/src/frameworks/angular/helpers.ts [hunk 2]
```diff
@@ -32,17 +32,32 @@ const createComponentFromTemplate = (template: string) => {
 };
 
 export const initModuleData = (storyObj: NgStory): any => {
-  const { component, template, props, moduleMetadata = {} } = storyObj;
+  const {
```
Comment (shilman) [inline addons/storyshots/storyshots-core/src/frameworks/angular/helpers.ts]: What's this doing here?

Comment (ndelangen) [reply addons/storyshots/storyshots-core/src/frameworks/angular/helpers.ts]: wtf

Comment (ndelangen) [reply addons/storyshots/storyshots-core/src/frameworks/angular/helpers.ts]: I dunno man, this seems to have sorted itself

**Review by ndelangen — APPROVED** [review]

**Pull request merged**